### DISCUSSION TITLE
Bluetooth repeater

### DISCUSSION THREAD

**Discussion by Nizdazen** [darthcloud/BlueRetro#931] [General]
Is there a bluetooth repeater that will work with the blueretro? The PowerA gamecube controller's range sucks and I was looking for a way to extend its range. My wavebirds have like double the range but Id like to try and have the aesthetics of the original gamecube controllers, which the PowerAs are closer to. I have cable extensions for the controllers but they are only 6 feet long and don't add much. Hoping for a wireless solution.

**Comment by darthcloud** [comment]
BT repeater aren't really a thing. How far are you? Maybe you could try orient the console in a different way or place it somewhere else to maximize signal strenght.

**Reply by Nizdazen** [reply]
Around 39 feet. I think its the controllers themselves because when I use ps3 controllers with blueretro (retroscaler) the range is more than double what I'm getting with n64 adapters (retrotime) combined with the Nintendo online n64 controller, or the powerA GameCube controller combined with the gamecube adapters (retrotime). The common factor seems to be Nintendo, and I know they like to cut corners (wii not having hdmi, n64 not have RGB etc.) I tried using some extension cables at one point for the consoles, but I'm currently upgrading all my systems to hd retrovision, it gets a bit harder with these much thicker cables. Now that my setup is upgraded to component/hdmi I need to figure out a different wireless video solution for now anyways.
